src/librustdoc/clean/types.rs

```diff
@@ -1624,9 +1624,20 @@ pub struct Impl {
 #[derive(Clone, Debug)]
 pub enum Import {
     // use source as str;
-    Simple(String, ImportSource),
+    // The bool indicates wether it imports a macro or not.
+    Simple(String, ImportSource, bool),
     // use source::*;
-    Glob(ImportSource),
+    // The bool indicates wether this is from an import.
```

Comment:
What do you mean by 'from an import'? Aren't all glob imports imports?

```suggestion
    // The bool indicates whether this is from an import.
```

Reply:
After your explanation in https://github.com/rust-lang/rust/pull/77276#discussion_r495999459 I think you meant 'inlined'.

Reply:
No, this is exactly what I meant. ;)